### DISCUSSION TITLE
How to trigger Twitter and Discord native apps from the Phantom Browser?

### DISCUSSION THREAD

**Discussion by shtefcs** [phantom/docs#141] [Help & Support]
### Summary

Hi guys, we recently released https://karamendos.com, where users go through the registration flow. In steps 2 and 3 of our flow, we are asking people to follow us on Twitter and join our Discord.

Our expectation from the optimal UX point of view is that those links should open native Twitter and Discord apps on both Android and IOS. 

We tried implementing the deep link/universal links feature but didn't see any change.

We tested from different devices, and from our testing, none of those devices was opening native apps. 

The problem is that its making very hard for users to interact with our registration flow because of the links opening inside of Phantom Browser.

### Example

https://karamendos.com

### Steps to Reproduce

Visit https://karamendos.com

1. Click **Join Party** button
2. Click **Get Started** button
3. Connect Wallet, Discord, Twitter
4. Click on Twitter and Discord links. On this step it should trigger native Twitter and Discord apps, but its actually opens in the same Phantom browser.

### Phantom Version

23.18

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hey @shtefcs,

Thank you for reporting this. We are investigating and will follow up once we have an update.

**Reply by shtefcs** [reply]
Thanks Adam. By your knowledge should this work by default? Like when Twitter and Discord links are clicked to be opened in their native apps.

**Reply by shtefcs** [reply]
How is going Adam?

**Reply by adamdelphantom** [reply]
Hi @shtefcs,

I don't have any updates. As soon as I have an ETA I will let you know. But feel free to check in anytime.

**Comment by sekosen** [comment]
Hi, are there any updates on this issue?

**Reply by adamdelphantom** [reply]
@mukkumayc No updates, yet. Thank you for checking in!
